### ISSUE TITLE
Irreführende Verwendung von MUST und optional

### ISSUE THREAD

**Issue by tliebscher** [hl7germany/basisprofil-de#118]
Wieso wird MUST verwendet, wenn es in der ursprünglichen VSDM Spezifikation um OPTIONAL ging?

https://simplifier.net/BasisprofilDE/patient-de-vsdm-0.2/~details
...
Patient Ressource im Kontext des deutschen VSDM (Versichertenstammdatenmanagement, "eGK-Datensatz"). Elemente, die als "must-support" gekennzeichnet sind, repräsentieren die **optionalen** Informationen aus der der VSDM-Spezifikation.

**Comment by patrick-werner** [comment]
Weil must-support keine Aussage über die Kardinalität ist, sondern ausdrückt dass ein System diese Elemente unterstützen muss.

**Comment by tliebscher** [comment]
Ja, richtig. Wäre die folgende Formulierung dann besser? Must-support repräsentiert hier ja nicht nur optionale Informationen (bezogen auf Kardinalität).

Patient Ressource im Kontext des deutschen VSDM (Versichertenstammdatenmanagement, "eGK-Datensatz"). Elemente, die als "must-support" gekennzeichnet sind, repräsentieren die ~~optionalen~~ Informationen aus der der VSDM-Spezifikation.

**Comment by simoneOnFhir** [comment]
Bitte berücksichtigen, wenn Patient-vsdm nach R4 migriert wird...
